package.py

```diff
@@ -1085,13 +1089,13 @@ def copy_file_to_target(file, temp_dir):
         pyproject_target_file = copy_file_to_target(pyproject_file, temp_dir)
 
         if os.path.isfile(poetry_lock_file):
-            log.info("Using poetry lock file: %s", poetry_lock_file)
+            log.info("Using poetry.lock file: %s", poetry_lock_file)
             poetry_lock_target_file = copy_file_to_target(poetry_lock_file, temp_dir)
         else:
             poetry_lock_target_file = None
 
         if os.path.isfile(poetry_toml_file):
-            log.info("Using poetry configuration file: %s", poetry_lock_file)
+            log.info("Using poetry.toml configuration file: %s", poetry_toml_file)
```

Comment:
This kind of duplicates the `poetry.toml` in the output, doesn't it?

```suggestion
            log.info("Using poetry toml configuration file: %s", poetry_toml_file)
```

Reply:
This was to be consistent with the `poetry.lock` file output  above, so if neither are desired, it can be removed.

Reply:
The larger change and bug fix was having the output point to `poetry_toml_file` instead of `poetry_lock_file`

Reply:
> The larger change and bug fix was having the output point to `poetry_toml_file` instead of `poetry_lock_file`

Yeah, noticed that part :+1: